File: tests/count_down_timer/py/count_down_timer.py
# countdown_timer.py
import time

def countdown(start):
    for i in range(start, -1, -1):
        print(f"Countdown: {i}")
        time.sleep(1)
    print("Time's up!")

if __name__ == "__main__":
    start = int(input("Enter the starting number for the countdown: "))
    countdown(start)

File: tests/count_down_timer/js/count_down_timer.js
function countdown(start) {
    let timer = setInterval(() => {
        console.log(`Countdown: ${start}`);
        start--;
        
        if (start < 0) {
            clearInterval(timer);
            console.log("Time's up!");
        }
    }, 1000);
}

let startNumber = parseInt(prompt("Enter the starting number for the countdown:"));
countdown(startNumber);